completion/available/nvm.completion.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# nvm (Node Version Manager) completion
+
+if [ -r $NVM_DIR/bash_completion ]
+then
+  . $NVM_DIR/bash_completion
+else
+  echo "Enable nvm plugin first"
```

Comment:
I think we shouldn't do this, I think it makes more sense to fail silently. We've had lots of error reports from people where their connectivity was impacted when the server they were connecting to was running Bash-it and messages were printed upon login. This was especially difficult for something like `scp`.

I'd prefer to let the completion fail silently if NVM is not installed...